watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -163,269 +111,99 @@ def adm(self):
-                    - adm.fs.unit.vapor_outlet.flow_vol[0] * adm.fs.props_vap.dens_mass
-                )
-            )
-            <= 1e-6
-        )
-
-        assert pytest.approx(-13.5835, abs=1e-3) == value(
-            adm.fs.unit.liquid_phase.enthalpy_transfer[0]
-        )
-        assert (
-            abs(
-                value(
-                    (
-                        adm.fs.unit.inlet.flow_vol[0]
-                        * adm.fs.props.dens_mass
-                        * adm.fs.props.cp_mass
-                        * (
-                            adm.fs.unit.inlet.temperature[0]
-                            - adm.fs.props.temperature_ref
-                        )
-                    )
-                    - (
-                        adm.fs.unit.liquid_outlet.flow_vol[0]
-                        * adm.fs.props.dens_mass
-                        * adm.fs.props.cp_mass
-                        * (
-                            adm.fs.unit.liquid_outlet.temperature[0]
-                            - adm.fs.props.temperature_ref
-                        )
-                    )
-                    + adm.fs.unit.liquid_phase.enthalpy_transfer[0]
-                )
-            )
-            <= 1e-2
-        )
-
-    @pytest.mark.solver
-    @pytest.mark.skipif(solver is None, reason="Solver not available")
-    @pytest.mark.component
-    def test_costing(self, adm):
-        m = adm
-
+        # Add unit model costing
         m.fs.costing = WaterTAPCosting()
 
         m.fs.unit.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
         m.fs.costing.cost_process()
-        m.fs.costing.add_LCOW(m.fs.unit.liquid_phase.properties_out[0].flow_vol)
-        solver = get_solver()
-        results = solver.solve(m)
-
-        assert_optimal_termination(results)
 
-        # Check solutions
-        assert pytest.approx(2.0 * 1083290.8, rel=1e-5) == value(
-            m.fs.unit.costing.capital_cost
+        # Set scaling factors for badly scaled variables
+        iscale.set_scaling_factor(
+            m.fs.unit.liquid_phase.mass_transfer_term[0, "Liq", "S_h2"], 1e7
         )
-        assert pytest.approx(5.2754, rel=1e-5) == value(m.fs.costing.LCOW)
+        iscale.set_scaling_factor(m.fs.unit.costing.capital_cost, 1e-6)
 
-    @pytest.mark.unit
-    def test_get_performance_contents(self, adm):
-        perf_dict = adm.fs.unit._get_performance_contents()
-
-        assert perf_dict == {
-            "vars": {
-                "Volume": adm.fs.unit.volume_AD[0],
-                "Heat Duty": adm.fs.unit.heat_duty[0],
-            }
+        self.unit_model_block = m.fs.unit
+        self.unit_statistics = {
+            "number_variables": 203,
+            "number_total_constraints": 171,
+            "number_unused_variables": 0,
         }
-
-    @pytest.mark.unit
-    def test_report(self, adm):
-        adm.fs.unit.report()
+        self.unit_solutions = ComponentMap()
```

Comment:
If we're going to require this to be a `ComponentMap`, I would suggest setting it (and the unit_statistics) up in `configure_class` before the call to `configure`.

Reply:
How would `self.unit_statistics` be defined here? `self.unit_statistics = ComponentMap()?`. This ties in with your earlier comment about the need to attach those objects to the block.

Reply:
No, it could just be a dictionary, or we could define a small class to hold the model statistics.

I don't really like checking these anyways. I don't know what it's suppose to catch, and it seems like something that has to be updated any time the model changes slightly -- even if the revised model is mathematically equivalent. (Thoughts @TimBartholomew?)

Reply:
Tim and I had discussed just getting rid of these tests entirely. Removing them would probably make everyones' lives a little easier with virtually no downside since the tests aren't very helpful in the first place.

Reply:
Yeah, please get rid of them. Checking the number of unused variables is good, but should probably just be done at the model development step and not in the tests.